src/scss/components/_fieldset.scss

```
@@ -23,4 +24,21 @@ fieldset {
       padding-bottom: var(--spacer-xx-small);
     }
   }
+
+  &.validation {
+    legend {
+      float: left;
+      width: 100%;
+      margin-top: 0;
+    }
+  }
+
+  &.fieldsetError {
```

```suggestion
  &.fieldset-error {
```
Follow naming convention